ami/models/components/vision_transformer_layer.py

```diff
@@ -0,0 +1,124 @@
+# Ref: https://github.com/facebookresearch/ijepa
+
+import math
+
+import numpy as np
+import numpy.typing as npt
+import torch
+import torch.nn as nn
+
+from .drop_path import DropPath
+
+
+class MLP(nn.Module):
+    """Multi Layer Perceptron."""
+
+    def __init__(
+        self,
+        in_features: int,
+        hidden_features: int,
+        out_features: int,
+        drop: float = 0.0,
```

Comment:
`drop`は動詞なので`dropout`と表記するのが良いです。
```suggestion
        dropout: float = 0.0,
```

Reply:
https://github.com/MLShukai/ami-vconf24/pull/40/commits/085b06ed95928eebdfc6fc4572c49fdf1c149981
で関係しそうな箇所を修正